spec.html

```diff
@@ -37093,7 +37093,7 @@ <h1>The %AsyncFromSyncIteratorPrototype% Object</h1>
         <ul>
           <li>has properties that are inherited by all Async-from-Sync Iterator Objects.</li>
           <li>is an ordinary object.</li>
-          <li>has a [[Prototype]] internal slot whose value is %AsyncIterator.prototype%.</li>
+          <li>has a [[Prototype]] internal slot whose value is %AsyncIteratorPrototype%.</li>
```

Comment:
maybe instead, we should make `%AsyncIterator%` an intrinsic?

Reply:
Maybe. Not my call. Until we do, referring to it is a bug.

Reply:
We can do that as an editorial change, perhaps in this PR.

Reply:
But isn't adding a new intrinsic a normative change?

Reply:
is it? it's not user-observable any way i can tell

Reply:
I guess if it's completely inaccessible then it wouldn't be user-observable.

Still, I'm not much interested in adding it.

BTW, I just noticed another: `%ArrayIterator.prototype%` assumes that `%ArrayIterator%` exists, but it doesn't.

Reply:
Let’s just revert these examples for now, and we can clean them up later.